marklogic-data-hub/src/test/java/com/marklogic/hub/flow/FlowRunnerTest.java

```diff
@@ -116,7 +117,13 @@ public void testRunFlow(){
     }
 
     @Test
-    public void testIngestCSVasXML(){
+    public void testIngestCSVasXML() throws Exception {
+        //prov docs cannot be read by "flow-developer-user", so creating a client using 'secUser' which is 'admin'
```

Comment:
Just a note that this will change after 5.1 when we introduce the data-hub-job-reader role.